### PR TITLE
refactor: 💡 Delete not necessary comments

### PR DESCRIPTION
**Pull request by calcaide**
# Description

Delete not necessary comments.

I do believe we do not need references to helpers we are not using. Its a pattern we are not following with any other package.
In need of knowing what helpers we have available, [just check package documentation](https://github.com/mainmatter/ember-simple-auth?tab=readme-ov-file#testing)

### CODE REVIEW

**Review by DhariniJeeva — APPROVED** [review]
thanks for the cleanup!

File: ui/admin/tests/acceptance/auth-methods/oidc-test.js [hunk 1]
```diff
@@ -4,24 +4,12 @@
...
-  click,
-  // fillIn,
-  find,
-  // findAll,
-} from '@ember/test-helpers';
+import { visit, currentURL, click, find } from '@ember/test-helpers';
 import { setupApplicationTest } from 'ember-qunit';
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 import a11yAudit from 'ember-a11y-testing/test-support/audit';
 //import { Response } from 'miragejs';
```
Comment (lisbet-alvarez) [inline ui/admin/tests/acceptance/auth-methods/oidc-test.js:11]:
```suggestion
```

**Review by lisbet-alvarez — APPROVED** [review]
looks great! thank u for cleaning this up!

**Review by cameronperera — APPROVED** [review]
👏🏼

**Pull request merged**